backports/festival/APKBUILD

```diff
@@ -0,0 +1,79 @@
+# Contributor: Atsushi Watanabe <atsushi.w@ieee.org>
+# Maintainer:
+pkgname=festival
+pkgver=2.5.0_git20180620
+pkgrel=0
+pkgdesc="Festival Speech Synthesis System"
+url="http://www.cstr.ed.ac.uk/projects/festival/"
+arch="all"
+license="custom:festival"
+depends=""
+makedepends="linux-headers ncurses-dev alsa-lib-dev git"
+subpackages="$pkgname-doc $pkgname-dbg"
+source="musl-support.patch
+  fix-libdir.patch
+  fix-optimization.patch
+  fix-memleaks.patch
+  add-nullptr-check.patch"
+options="!check"  # test is not provided
+
+builddir="$srcdir"
+
+prepare() {
+  cd "$srcdir"
+  git clone https://github.com/festvox/festival.git
+  git -C festival checkout 1330e5350e
+  git clone https://github.com/festvox/speech_tools.git
+  git -C speech_tools checkout b8b5568e03
+  default_prepare
+}
+
+build() {
+  cd "$builddir"/speech_tools
+  ./configure
+  make \
+    CFLAGS="-fPIC -g -Os -fno-strict-aliasing" \
```

Comment:
It still has upstream bugs on c++ class usage. Debug info can be installed by `festival-dgb` package.